scripts/verify-bun-review-artifacts.py

```
@@ -2036,6 +2036,469 @@ def require_joined_revision(value, label: str, binding: dict) -> None:
+def _task_ledger_source(value) -> object:
+    if isinstance(value, str) and value in {
+        "Required", "Configured", "Impact", "Sensor", "Worker"
+    }:
+        return value
+    if isinstance(value, dict) and set(value) == {"ReviewerTurn"}:
+        payload = value["ReviewerTurn"]
+        _task_ledger_strict_fields(payload, {"model_on"}, "reviewer-turn source")
+        if not isinstance(payload["model_on"], bool):
+            fail("[unsupported_schema] reviewer-turn model_on must be boolean")
+        return {"ReviewerTurn": {"model_on": payload["model_on"]}}
+    fail(f"[unsupported_schema] task source is invalid: {value!r}")
```

Ruff (0.16.2)

[warning] Missing explicit `return` at the end of function able to return non-`None` value

Add explicit `return` statement

(RET503)